src/hotspot/share/memory/universe.cpp

```diff
@@ -1098,7 +1093,6 @@ bool universe_post_init() {
      tty->print_cr("Unable to link/verify VirtualMachineError class");
      return false; // initialization failed
   }
```

Comment:
Do we still need to link VirtualMachineError above?

Reply:
Good question.  I don't know if we have to link and initialize it.  We need the Klass because there are subclass checks.

Reply:
I put this in the commit comment.  We don't need to explicitly do anything with VME in universe because the code just below this will link and initialize and create an instance of StackOverflowError and that'll link VME as a super class.